Makefile

```diff
@@ -280,3 +280,14 @@ mixins:
 	$(CONTAINER_ENGINE) pull registry.svc.ci.openshift.org/ci/dashboards-validation:latest
 	$(CONTAINER_ENGINE) run --user=$(UID) --rm -v "$(CURDIR):/release:z" registry.svc.ci.openshift.org/ci/dashboards-validation:latest make -C /release/clusters/app.ci/prow-monitoring/mixins install all
 .PHONY: mixins
+
+# Runs e2e secrets generation and sync to clusters.
+#
+# Example:
+#   make secrets
+# First execute the following
+# echo -n "bw_password" > /tmp/bw_password
+# make kerberos_id=<your_kerberos_id> secrets
+secrets:
+	hack/secrets.sh $(kerberos_id)@redhat.com $(kubeconfig_path)
```

Comment:
You must not append the `@redhat.com` here, its already appended in the `ci-secret-generator` make target